src/MainRouter.tsx

```diff
@@ -96,8 +98,15 @@ const MainRouter = (): ReactElement => {
       <Route path={ROUTES.GAMIFICATION} element={<Gamification />} />
       <Route path={ROUTES.AGENDA} element={<Agenda sessions={sessions} />} />
       <Route path={ROUTES.ROOMS} element={<RoomsPage />} />
+      <Route path={"/chat_dev"} element={<Chat
+          topic={"gsoc-11"}
+          privKey={"0x527f11716334d9565179db07bca7de808bda1be8456d00975045ce40b9abf5bb"}
+          stamp={"03263b84b1263899b4e63ea6b2ce4fb252c280a31a128b5d54f44cd0807c9e0e" as BatchId}
+          nickname={"Peter"}
+          gsocResourceId={"f6b7beefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef"}
```

Comment:
TODO: make these variable.

Reply:
This will totally go out from here later, the topics like "Layer 2", "Security", will call this component. I can make these variables, but this is just a mock.